File: githublink.py
import time
from selenium import webdriver

if __name__ == '__main__':

    # Chrome WebDriver 로 chrome.exe 실행
    options = webdriver.ChromeOptions()

    # 브라우저 안 띄우기
    # options.add_argument("headless")

    # 그래픽 카드 사용 안하기
    options.add_argument("disable-gpu")

    # 브라우저 크기 지정
    options.add_argument("window-size=1920,1080")

    browser = webdriver.Chrome('./chromedriver.exe', options=options)

    # 사이트 호출
    browser.implicitly_wait(10)
    browser.get("https://github.com/ImDaeseong?tab=repositories")

    # DaeseongSwift link 페이지 호출
    browser.implicitly_wait(10)
    browser.find_element_by_link_text('DaeseongSwift').click()

    # 페이지 읽기
    # print(browser.page_source)

    # 3초후 종료
    time.sleep(3)

    # 드라이브 종료
    browser.quit()

File: golang/githublink.go
// githublink
package main

import (
	"fmt"
	"time"

	"github.com/tebeka/selenium"
)

const (
	path = `E:\GoApp\src\chromedriver.exe`
	port = 8080
)

func main() {

	opts := []selenium.ServiceOption{}

	server, err := selenium.NewChromeDriverService(path, port, opts...)
	if err != nil {
		fmt.Println(err)
	}
	defer server.Stop()

	c := selenium.Capabilities{"browserName": "chrome"}

	wd, err := selenium.NewRemote(c, fmt.Sprintf("http://localhost:%d/wd/hub", port))
	if err != nil {
		fmt.Println(err)
	}
	defer wd.Quit()

	err = wd.Get("https://github.com/ImDaeseong?tab=repositories")
	if err != nil {
		panic(err)
	}

	wd.SetImplicitWaitTimeout(10 * time.Second)

	link, _ := wd.FindElement(selenium.ByLinkText, "DaeseongSwift")
	link.Click()

	wd.SetImplicitWaitTimeout(10 * time.Second)

	url, _ := wd.CurrentURL()
	title, _ := wd.Title()
	pagesource, _ := wd.PageSource()

	fmt.Println(url)
	fmt.Println(title)
	fmt.Println(pagesource)

	time.Sleep(10 * time.Second)
}
